e2e/__tests__/stackTraceSourceMapsWithCoverage.test.ts

```diff
@@ -7,6 +7,7 @@
 import * as path from 'path';
 import {extractSummary, run} from '../Utils';
 import runJest from '../runJest';
+import wrap from 'jest-snapshot-serializer-raw';
```

Comment:
eslint will want this sorted alphabetically 😬 just `eslint --fix e2e/__tests__/stackTraceSourceMapsWithCoverage.test.ts`

Reply:
whoops. fixed this.